readthedocs/oauth/tasks.py

```diff
@@ -22,20 +31,12 @@
 
 
 @permission_check(user_id_matches)
```

Comment:
In the celery docs, it says that we shouldn't do this (put the task decorator down), but the task decorator overrides all function's attributes and deletes the added by `permission_check`, this way we can keep the added attribute.